hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/insights.tsx

```diff
@@ -36,10 +37,17 @@ interface IFileCountResponse {
   count: number;
 }
 
+interface IContainerCountResponse {
+  containerSize: number;
+  count: number;
+}
+
 interface IInsightsState {
   isLoading: boolean;
   fileCountsResponse: IFileCountResponse[];
+  containerCountResponse: IContainerCountResponse[];
   plotData: Plotly.Data[];
+  containerData: Plotly.Data[];
```

Comment:
as we are having both container and file count size task now, we should rename the previous variable, so it is easier to understand it. 
```suggestion
  fileCountData: Plotly.Data[];
  containerCountData: Plotly.Data[];
```

Reply:
@dombizita 
Completed Change Suggested by you.